tracing-core/src/dispatch.rs

```diff
@@ -859,6 +884,63 @@ where
     }
 }
 
+impl WeakDispatch {
+    /// Attempts to upgrade the `WeakDispatch` to a [`Dispatch`].
+    ///
+    /// Returns `None` if the inner `Dispatch` has already been dropped.
```

Comment:
```suggestion
    /// Attempts to upgrade this `WeakDispatch` to a [`Dispatch`].
    ///
    /// Returns `None` if the referenced `Dispatch` has already been dropped.
```

Reply:
also, take it or leave it, but it might be worth saying something like "this is analogous to `Weak::upgrade` in the docs here, as well?